packages/uikit-react-native/src/localization/createBaseStringSet.ts

```diff
@@ -261,6 +263,7 @@ export const createBaseStringSet = ({ dateLocale, overrides }: StringSetCreateOp
       CHANNEL_MESSAGE_EDIT: 'Edit',
       CHANNEL_MESSAGE_SAVE: 'Save',
       CHANNEL_MESSAGE_DELETE: 'Delete',
+      CHANNEL_MESSAGE_REPLY: (user) => (user ? `Reply to ${user.nickname}` : 'Reply'),
```

Comment:
요기엔 그냥 Reply 텍스트로 되어있는거 같습니다
```suggestion
      CHANNEL_MESSAGE_REPLY: 'Reply',
```